article/app/views/liveblog/liveBlogBlock.scala.html

```diff
@@ -3,7 +3,7 @@
 @import model.liveblog.{LiveBlogDate, BodyBlock}
 @import views.BodyProcessor
 @import model.DotcomContentType
-@(block: BodyBlock, article: Article, timezone: DateTimeZone, isPinnedBlock: Boolean = false)(implicit request: RequestHeader, context: model.ApplicationContext)
+@(block: BodyBlock, article: Article, timezone: DateTimeZone, isPinnedBlock: Boolean = false, isPinnedBlockInSitu: Boolean = false)(implicit request: RequestHeader, context: model.ApplicationContext)
```

Comment:
I wonder if there's a better name for this. Its a tricky one to name for sure! Would `isOriginalPinnedBlock` be clearer? I'm not really sure so keen to hear thoughts. Happy to keep with what is here but it feels like the phrase `in situ` has thrown a couple of people already.

Reply:
Actually I myself didn't even know what it meant 😃  I'm happy with the name you suggested `isOriginalPinnedBlock`
Does anyone else have any suggestion for this naming? @alinaboghiu @joecowton1

Reply:
I think that new suggestion is probably good, it makes it easier to understand anyway :)